File: POO_Uber/Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car {
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterials;

    public UberVan(String license, Account driver, Map<String, 
    Map<String,Integer>> typeCarAccepted,
    ArrayList<String> seatsMaterials){

        super(license,driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterials = seatsMaterials;
    }
}

File: POO_Uber/Python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccept = list
    seatsMaterials = list

    def __init__(self, license, drive, typeCarAccept, seatsMaterials):
        super().__init__(license, drive)
        self.typeCarAccept = typeCarAccept
        self.seatsMaterials = seatsMaterials